src/scss/styles.scss

```diff
@@ -1,20 +1,51 @@
 @import "bourbon";
 @import "neat";
 
+@import url(http://fonts.googleapis.com/css?family=Raleway:400,600,700,300);
+@import url(http://fonts.googleapis.com/css?family=Roboto:400,600,700,300);
+

```

Comment:
In the master branch I'd like to keep the number of fonts down. I think these belong in each users' defined style for their site.